02-express-tutorial/app.js

```diff
@@ -1,35 +1,36 @@
 const express = require('express');
-const {products} = require('./data');
-const app = express();
+const cookieParser = require('cookie-parser');
```

Comment:
You `require` this package, but you don't have it pushed up to GitHub from your `package.json`!

Doing `npm install cookie-parser` locally let me work around this.